lib/sqlalchemy/dialects/mssql/base.py

```diff
@@ -938,9 +938,11 @@ class MyClass(Base):
 from ...sql import quoted_name
 from ...sql import roles
 from ...sql import sqltypes
+from ...sql import try_cast as try_cast  # noqa: F401
 from ...sql import util as sql_util
```

Comment:
**Nick Crews** (NickCrews) wrote:

We do need the noqa still (per `pre-commit run --files lib/sqlalchemy/dialects/mssql/base.py`, not sure if this is the canonical way to lint).

But I think you might be right that I think there is some config in flake8 that we could turn on so this wasn't needed.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4605